### ISSUE TITLE
Errors are ignored with multiple script blocks 

### ISSUE THREAD

**Issue by chenqiushi** [vuejs/eslint-plugin-vue#538]
**Tell us about your environment**

* **ESLint Version:**
5.2.0
* **eslint-plugin-vue Version:**
5.0.0-beta.0
* **Node Version:**
8.4.0

**Please show your full configuration:**

```
module.exports = {
  extends: [
    'plugin:vue/recommended'
  ],
  rules: {
    'vue/script-indent': 'error'
  }
}
```

**What did you do? Please include the actual source code causing the issue.**

```vue
<template>
  <div>foo bar baz</div>
</template>

<style scoped>
  div {
    color: red;
  }
</style>

<script src="xxx"></script>
<script>
export default {
  mounted () {
  console.log('wrong indent here')
  }
}
</script>

```

**What did you expect to happen?**

It is suggested that each *.vue file can contain at most one <script> block at a time. I wrote two <script> blocks here by mistake, so I expect there will be errors about this ?

Even if the above rule is ignored, indent error of the second <script> block is expected: 

eslint will throw error:
error  Expected indentation of 4 spaces but found 0 spaces  vue/script-indent

**What actually happened? Please include the actual, raw output from ESLint.**
The indent error was ignored and there was no output.

**Comment by mysticatea** [comment]
Thank you for this issue.

Yes.
This plugin checks only the first `<script>` element.
But if `.vue` files allow multiple `<script>` elements, this plugin should check those.
What does it happen if `export default` exists multiple?

And I'd like to ask, can't you use `import from "./xxx"` in that case?

**Comment by chenqiushi** [comment]
If `export default` exists multiple times and I use `import xxx from "path/xxx"` in another component, only the last `<script>` will be exported. As you said, this plugin checks only the first `<script>` element, but now the last `<script>` is exported and available... I guess should we only check the last one ?

**Comment by michalsnik** [comment]
This could be fixed in the `vue-eslint-parser`
